content/waku/2024-gantt.md

```diff
@@ -0,0 +1,180 @@
+# Waku Roadmap 2024 Gantt Charts
```

Comment:
general question about definitions:
- if a `day` is a working day;
- then `week` is `5d` or `7d`?

Reply:
estimates in week, meaning 5 days of work is a week.